frontend/src/pages/Login.vue

```diff
@@ -313,7 +303,7 @@ If you're able to authenticate against the Key Store and Private Key, your Key S
                     v-if="!this.$store.state.app.isLoggedIn"
                     type="info"
                     block
-                    :disabled="!this.$store.state.validators.valid_password || !this.$store.state.validators.storepass.valid_password || this.loginInProgress || this.alias === '' || this.keystorePassword === '' || this.keyPasswordValidate === ''"
+                    :disabled="invalid_new_wallet"
```

Comment:
This seems to work better. :smile_cat: 
```suggestion
                    :disabled="!this.valid_new_wallet"
```